libsolutil/CommonData.h

```diff
@@ -343,6 +343,49 @@ void joinMap(std::map<K, V>& _a, std::map<K, V>&& _b, F _conflictSolver)
 	}
 }
 
+template<typename T>
+class UniqueVector
+{
+public:
+	std::vector<T>& contents() { return m_contents; }
```

Comment:
Wasn't the point of having to implement `swap()` and all the extra helpers that it allows us to keep acccess to the content const?

```suggestion
	std::vector<T> const& contents() const { return m_contents; }
```